app/models/user.rb

```diff
@@ -114,13 +114,9 @@ def active_for_authentication?
   end
 
   def inactive_message
-    if active? && enabled?
-      super
-    elsif softly_deleted?
-      'This account has been deleted.'
-    elsif disabled?
-      'This account has been disabled.'
-    end
+    return 'Invalid Email or password.' unless active? && enabled?
```

Comment:
Could this message go in a translation file rather than be hardcoded here? It would be a little neater and might make it easier to maintain in future.